src/bundle/Resources/public/scss/_custom.scss

```
@@ -28,6 +28,7 @@ $ibexa-color-font: #131c26;
 $ibexa-color-font-light: #fff;
 $ibexa-color-font-pale: #878b90;
 $ibexa-color-font-black: #222222;
+$ibexa-color-font-danger:#d92d42;
```

I am really sorry, but... :D 
```suggestion
$ibexa-color-font-danger: #d92d42;
```